src/main/perf/LineFileDocs.java

```diff
@@ -155,7 +153,7 @@ private void readDocs() throws Exception {
         if (x != 8) {
           throw new RuntimeException("expected 8 header bytes but read " + x);
         }
-        int count = header.getInt(0);
+        int docCountPerBlock = header.getInt(0);
```

Comment:
Maybe rename to `docCountInBlock`?  Since it changes from block to block?

Reply:
++